pom.xml

```diff
@@ -632,7 +632,7 @@
         <jackson.version>2.9.6</jackson.version>
         <jetty.version>9.4.7.v20170914</jetty.version>
         <testcontainers.version>1.8.1</testcontainers.version>
-        <metrics.version>3.2.6</metrics.version>
+        <metrics.version>4.0.3</metrics.version>
```

Comment:
upgrading dropwizard metrics leads to incompatible metrics version in cassandra driver dependency, see ci build error in https://github.com/linagora/james-project/pull/1560
Neither you declare old metric dependency go with cassandra driver dependency nor drop the metrics upgrade.

mvn dependency tree result:
```
[INFO] +- com.datastax.cassandra:cassandra-driver-core:jar:shaded:3.5.1:compile
[INFO] |  +- io.dropwizard.metrics:metrics-core:jar:3.2.6:compile
[INFO] |  +- com.github.jnr:jnr-ffi:jar:2.1.7:compile
```

Reply:
Let's keep metrics at 3.2.6 IMO